src/style.css

```
@@ -1 +1,75 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color-link-active: #00acdc;
+  --color-link: #000;
+  --color-background-header: #fff;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+  padding: 0 50px;
+  background: var(--color-background-header);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  position: relative;
+  align-items: center;
+  text-transform: uppercase;
+  margin-left: 20px;
+  padding: 10px 0;
+  font-weight: 500;
+  height: 60px;
+  font-size: 12px;
+  line-height: 14px;
+  color: var(--color-link);
+}
+
+.nav__link:hover {
+  color: var(--color-link-active);
+}
+
+.is-active {
+  margin: 0;
```

the same
you can simplify
```suggestion
.nav__link:hover, .is-active {
  color: var(--color-link-active);
}
```